.gitignore

```diff
@@ -19,3 +19,4 @@ typedef.list
 **/GRTAGS
 **/GSYMS
 /.vs
+compile_commands.json
```

Comment:
what is this? Is this JSON file just something on your local computer?

Reply:
No, it is the file that you generate through
 `./bootstrap -DCMAKE_EXPORT_COMPILE_COMMANDS=YES` in order to make sure VSC through cquery recognizes variables in C and provides easy jumping around features.
I thought because we all are using cquery and vsc now, it would be good not to see this file in every `git status`